PixelGamEngineSolution/FlappyBird/Bird.h

```diff
@@ -0,0 +1,106 @@
+#pragma once
+#include "olcPixelGameEngine.h"
+
+
+class FBird
+{
+private:
+	olc::vf2d Position = {0, 0};
+
+	float VelocityY = 0.0f;
+	float AccelerationY = 0.0f;
```

Comment:
I think there's no need for the Ys at the end, there is only ever velocity and acceleration with respect to Y so it's implicit